### PR TITLE
Update multi-dataset benchmark definition

### PR DESCRIPTION
**Pull request by glpuga**
### Proposed changes

Version of the report associated to the post in https://gitlab.com/ekumenlabs/ekumen-website/-/merge_requests/447 .

This update is post-fixes to the maps in the TorWIC SLAM dataset, and the drive type of the Willow dataset.

#### Type of change

- [x] 🐛 Bugfix (change which fixes an issue)
- [ ] 🚀 Feature (change which adds functionality)
- [ ] 📚 Documentation (change which fixes or extends documentation)

💥 **Breaking change!** _Explain why a non-backwards compatible change is necessary or remove this line entirely if not applicable._

### Checklist

_Put an `x` in the boxes that apply. This is simply a reminder of what we will require before merging your code._

- [ ] Lint and unit tests (if any) pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### Additional comments

_Anything worth mentioning to the reviewers._

### CODE REVIEW

File: src/benchmarks/beluga_vs_nav2_multi_dataset/reports/nominal_report/conf.py [hunk 1]
```diff
@@ -23,11 +23,11 @@
...
 
 project = 'Nominal Beluga AMCL Benchmark Report'
-copyright = '2023, Ekumen Inc.'
+copyright = '2024, Ekumen Inc.'
 author = 'Ekumen Inc.'
 
-version = '0.1.0'
-release = '0.1.0-alpha'
+version = '0.1.1'
+release = '0.1.1'
```
[the remark this answers is not in the document]
Comment (glpuga) [reply src/benchmarks/beluga_vs_nav2_multi_dataset/reports/nominal_report/conf.py:30]: I would say that 0.1.0 was the first report were we idetified the issues with Willow and TorWIC, leading to this new iteration, but as long as **public** report versions are numbered differently, I don't think it matters (we also have dates to tell document versions apart).

[the remark this answers is not in the document]
Comment (glpuga) [reply src/benchmarks/beluga_vs_nav2_multi_dataset/reports/nominal_report/conf.py]: Changed it to 0.1.0

File: src/benchmarks/beluga_vs_nav2_multi_dataset/scripts/nominal.robot [hunk 2]
```diff
@@ -97,76 +97,74 @@ ${WILLOW_DSET_DIR}           willow_garage_dataset_ros2_localization_only
...
+TorWIC Mapping            ${{TORWIC_MAPPING_BAGS}}        ${{TORWIC_MAPPING_DIR}}   ${{LASER_MODELS}}     odom           map        base_link       /front/scan    1              0.0             0.0             0.0                nav2_amcl::DifferentialMotionModel
+TorWIC SLAM               ${{TORWIC_SLAM_BAGS}}           ${{TORWIC_SLAM_DIR}}      ${{LASER_MODELS}}     odom           map        base_link       /front/scan    1              0.0             0.0             0.0                nav2_amcl::DifferentialMotionModel
+Willow Garage             ${{WILLOW_DSET_BAGS}}           ${{WILLOW_DSET_DIR}}      ${{LASER_MODELS}}     odom_combined  map        base_footprint  /base_scan     1              0.0             0.0             0.0                nav2_amcl::OmniMotionModel
+# Diff Drive Sim 24hs       ${{DIFF_DRIVE_SIM_BAGS}}        ${{LONG_DURATION_DIR}}    ${{LASER_MODELS}}     odom           map        base_link       /scan          1              0.0             2.0             0.0                nav2_amcl::DifferentialMotionModel
```
[the remark this answers is not in the document]
Comment (glpuga) [reply src/benchmarks/beluga_vs_nav2_multi_dataset/scripts/nominal.robot]: While I won't remove the dataset just yet, I removed the commented lines for it in the nominal.robot script.

**Comment by glpuga** [comment]
@hidmic I did some small refactoring to section 1.2 and changed the title of the report in [ab987c4](https://github.com/Ekumen-OS/lambkin/pull/117/commits/ab987c4ef8f172b158fb39d0f61d67276a62abcf) . 

This is the current rc:  [report-draft.pdf](https://github.com/user-attachments/files/17979568/report-draft.pdf)

I think we are good to publish.

**Review by hidmic — APPROVED** [review]
go go go :rocket:

**Pull request merged**